build/serve.js

```diff
@@ -79,6 +79,10 @@ function getBackendArgs(mode) {
     args.push(`--apiserver-host=${conf.backend.envApiServerHost || conf.backend.apiServerHost}`);
   }
 
+  if (!conf.backend.disableSkipButton) {
```

Comment:
since `undefined` is falsy, would a missing config value re-enable the skip button? that seems fragile

Reply:
This is only used by our development pipeline and can't actually affect anyone.